jest-preprocess.js

```diff
@@ -0,0 +1,5 @@
+const babelOptions = {
```

Comment:
could we move this file into a new folder `src/testutil/jest-preprocess.js`

I prefer not to have too many files on the root folder